model/flow/payload.go

```diff
@@ -6,11 +6,16 @@ import (
 
 // Payload is the actual content of each block.
 type Payload struct {
-	// Guarantees are ordered in execution order.
+	// Guarantees are ordered in execution order. May be empty, in which case
+	// only the system chunk is executed for this block.
 	Guarantees []*CollectionGuarantee
-	Seals      []*Seal
-	Receipts   ExecutionReceiptMetaList
-	Results    ExecutionResultList
+	// Seals contains the block seals, for ancestor blocks, included in this payload.
+	// The oldest seal must connect to the latest seal in the fork extended by this block.
+	// Seals must be internally connected, containing no seals with duplicate block IDs or heights.
+	// Seals may be empty.
+	Seals    []*Seal
```

Comment:
Let's mention that seals are not ordered?

Reply:
Of course, the entire reason I was updating the godoc 🤦. Added in https://github.com/onflow/flow-go/pull/3461/commits/61fbabc1c79d49ab7e530f656e005547a1779696